Cargo.toml

```diff
@@ -40,7 +40,29 @@ members = [
     "crates/test-utils",
     "crates/workspace-hack",
     "crates/x",
-    "narwhal/config", "narwhal/consensus", "narwhal/crypto", "narwhal/dag", "narwhal/examples", "narwhal/executor", "narwhal/network", "narwhal/node", "narwhal/primary", "narwhal/storage", "narwhal/test-utils", "narwhal/types", "narwhal/worker",
+    "mysten-infra/crates/component", 
```

Comment:
Can we use just one directory level for crates, e.g. `mysten/` or `infra/`? Using `crates/` as suggested would also work, but it will make fixing the Dockerfiles harder.